### ISSUE TITLE
Header comment gets reapplied

### ISSUE THREAD

**Issue by BackEndTea** [FriendsOfPHP/PHP-CS-Fixer#3467]
For configuration or updating questions please read the README and UPGRADE documentation,
or visit: https://gitter.im/PHP-CS-Fixer

When reporting an issue (bug) please provide the following information:

#### The PHP version you are using (`$ php -v`):
=> PHP 7.1.13 (cli) (built: Jan  3 2018 08:27:24) ( NTS )
Copyright (c) 1997-2017 The PHP Group
Zend Engine v3.1.0, Copyright (c) 1998-2017 Zend Technologies

#### PHP CS Fixer version you are using (`$ php-cs-fixer -V`):
=> PHP CS Fixer 2.10.0 Bowling Bear by Fabien Potencier and Dariusz Ruminski (429e3a2)

#### The command you use to run PHP CS Fixer:
=> ./php-cs-fixer-v2.phar fix --config=.php_cs.dist -v --allow-risky=yes"

#### The configuration file you are using, if any:
```php
<?php

$year = date('Y');
$header = <<<TXT
Copyright © 2017-$year Maks Rafalko

License: https://opensource.org/licenses/BSD-3-Clause New BSD License
TXT;
$finder = \PhpCsFixer\Finder::create()
   ->in('src')
   ->in('tests')
   ->exclude('Fixtures')
   ->in('app')
;

return \PhpCsFixer\Config::create()
    ->setRules([
        '@Symfony' => true,
        '@PHP70Migration:risky' => true,

        'concat_space' => ['spacing' => 'one'],
        'header_comment' => [
            'commentType' => 'PHPDoc',
            'header' => $header,
            'location' => 'after_open',
            'separate' => 'bottom',
        ],
        'is_null' => true,
        'phpdoc_align' => false,
        'phpdoc_summary' => false,
        'yoda_style' => false,
    ])
    ->setFinder($finder)
;

```

#### If applicable, please provide minimum samples of PHP code (as plain text, not screenshots):
 * before running PHP CS Fixer (no changes):
```php
<?php
/**
 * Copyright © 2017-2018 Maks Rafalko
 *
 * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
 */

namespace Infection\Tests\Filesystem\Exception;

```

 * with unexpected changes applied when running PHP CS Fixer:
```php
<?php
/**
 * Copyright © 2017-2018 Maks Rafalko
 *
 * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
 */

declare(strict_types=1);
/**
 * Copyright © 2017-2018 Maks Rafalko
 *
 * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
 */

namespace Infection\Tests\Filesystem\Exception;
```

 * with the changes you expected instead:
```php
<?php
/**
 * Copyright © 2017-2018 Maks Rafalko
 *
 * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
 */

declare(strict_types=1);

namespace Infection\Tests\Filesystem\Exception;
```
The header comment gets applied again after the declare strict, for files where the declare strict was not previously present.

If the file contained no header comment, and no declare strict, they are added in the right order.

**Comment by k00ni** [comment]
Whats the status here?

**Comment by julienfalque** [comment]
This issue is still waiting for someone to pick it up. If one wants to give it a try, it should be done as [discussed in #3468](https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3468#issuecomment-373523191) IMO.

**Comment by k00ni** [comment]
@keradus: Can you tell me, if this issue was fixed by @4895 ?

**Comment by julienfalque** [comment]
@k00ni Yep, this issue was closed because #4895 was merged.
